test/parallel/test-common.js

```diff
@@ -108,3 +108,23 @@ const HIJACK_TEST_ARRAY = [ 'foo\n', 'bar\n', 'baz\n' ];
   common[`restoreStd${txt}`]();
   assert.strictEqual(originalWrite, stream.write);
 });
+
+// hijackStderr and hijackStdout again
+// for console
+[ 'err', 'out' ].forEach((txt) => {
+  common[`hijackStd${txt}`](common.mustCall(function(data) {
+    assert.strictEqual(data, 'test\n');
+
+    // throw an error
+    throw new Error(`console ${txt} error`);
+  }));
+
+  console[txt === 'err' ? 'error' : 'log']('test');
+  common[`restoreStd${txt}`]();
+});
+
+let uncaughtTimes = 0;
+process.on('uncaughtException', common.mustCallAtLeast(function(e) {
```

Comment:
How is the `mustCallAtLeast` fit with the `${([ 'err', 'out' ])[uncaughtTimes++]}` trick?